src/signing/networkMessages/kava-testnet.js

```diff
@@ -0,0 +1 @@
+export * from "./cosmos-hub-mainnet.js"
```

Comment:
we should simplify and define a `network_message_constructor` property in the networks object. This way we can add the networks without needing to set these in most cases

Reply:
Also reduced client side code

Reply:
oh yeah, good idea!